edx-platform/utec/lms/templates/instructor/edx_ace/allowedunenroll/email/body.html

```diff
@@ -0,0 +1,43 @@
+{% extends 'ace_common/edx_ace/common/base_body.html' %}
+
+{% load i18n %}
+{% load static %}
+{% block content %}
+<table width="100%" align="left" border="0" cellpadding="0" cellspacing="0" role="presentation">
+    <tr>
+        <td>
+            <h1>
+                {% blocktrans %}You have been unenrolled from {{ course_name }}{% endblocktrans %}
+            </h1>
+
+            <p style="color: rgba(0,0,0,.75);">
+                Hola,
+                <br />
+            </p>
+
+            <p style="color: rgba(0,0,0,.75);">
+                Has sido desinscripto del curso “{{ course_name }}” por un moderador. Este curso ya no aparecerá en tu panel principal.
```

Comment:
typo: desinscripto

Reply:
not a typo, that is how they say it in some places @jfavellar90 it was like that in UTEC requirements

Reply:
La verdad no tenía idea que las dos formas eran correctas: https://www.rae.es/dpd/inscribir. Ahora soy un poco menos bruto